src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html

```diff
@@ -0,0 +1,18 @@
+<kirby-list [items]="items"
+        shape="rounded">
```

Comment:
Shape is now rounded by default:
```suggestion
        >
```